src/twister2/builder/cmakecache.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+# vim: set syntax=python ts=4 :
+#
+# Copyright (c) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import re
+from collections import OrderedDict
+
+class CMakeCacheEntry:
+    '''Represents a CMake cache entry.
```

Comment:
Can you replace single quotes with double quotes in all places to be consistent with the rest of the code? (Also PyCharm says triple double-quotes should be use for docstrings).

Reply:
I did this draft only to share with you the idea of applying KConfig and DTS filtration. I copied several files 1:1 from Twister v1. This draft is not ready for review.